main.go

```diff
@@ -186,14 +189,14 @@ func summaryMessage(base, head int) string {
 	return fmt.Sprintf("Coverage increased by `%.2f%%`. :medal_sports: Keep it up :medal_sports:", float64(head-base)/100)
 }
 
-func getModulePackageName() string {
+func getModuleName() string {
```

Comment:
It seems like this is a misnomer? Perhaps `getPackageName()` is better if we want to shorten it?